log of add contributors and their team fixes

bharatagsrwal opened the pull request:
Forget to add whether a contributor includes that team in which they actually contributed, which fixes in this pull request, Now getting more accurate results.

in continuation of pull request [https://github.com/HTTPArchive/almanac.httparchive.org/pull/1658](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1658)

rviscomi commented:
Nice! Could you share the output?

bharatagsrwal commented:
> Nice! Could you share the output?

sure @rviscomi,

before
![image](https://user-images.githubusercontent.com/43988371/101586849-cd184000-3a08-11eb-93c0-06aadcccdf36.png)

after
![image](https://user-images.githubusercontent.com/43988371/101586691-7b6fb580-3a08-11eb-8f36-215402dbe92d.png)

rviscomi commented:
Looks good. Could you apply those recommended changes in this PR as well, effectively silencing the warnings?

bharatagsrwal commented:
> Looks good. Could you apply those recommended changes in this PR as well, effectively silencing the warnings?

Should I do with 2020 only, or 2019 also?

bharatagsrwal commented:
after changing `2019.json` and `2020.json`, No warning

![image](https://user-images.githubusercontent.com/43988371/101587709-c4c10480-3a0a-11eb-8f03-82e2eee73b79.png)

bharatagsrwal commented:
`AAgar`, `cheneytsai` and `malchata` haven't contributed to any of the team as a part of, so I removed them and others contributed as other teams also as they are not part of so I added those team in which they contributed.

rviscomi commented:
Thanks @bharatagsrwal!
